recipes/libiec61850/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy, rm, rmdir
+from os.path import join
+
+required_conan_version = ">=2.1"
+
+class Libiec61850Conan(ConanFile):
+    name = "libiec61850"
+    description = "An open-source library for the IEC 61850 protocols."
+    license = "LGPL-3.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libiec61850.com/libiec61850"
+    topics = ("iec61850", "mms", "goose", "sampled values")
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    languages = ["C"]
+
+    def layout(self):
+        cmake_layout(self)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
```

Comment:
Upstream CMakeLists.txt seems to be vendorizing a few dependencies - this would be an issue when using the recipe within a graph that shares them from CCI, we'll need to check if those can be made to come from Conan

Reply:
Thanks for the feedback!
Hmm, since the search for some optional dependencies is explicitly hardcoded into the CMake files, we don't have much flexibility to modify it right now to fetch them from CCI. Given this, the library will be built without TLS and log service support by default.